src/_media-queries.css

```diff
@@ -0,0 +1,3 @@
+@custom-media --breakpoint-not-small screen and (min-width: 30em);
```

Comment:
The only thing that concerns me about this is what happens if/when they depreciate the `not-small` breakpoint. If they just update the `medium` breakpoint to be exactly like the `not-small` then I think we're fine right?